### ISSUE TITLE
server blocked

### ISSUE THREAD

**Issue by mudcarofficial** [w180112/RDMA-example#4]
the function _prepare_recv_notify_before_using_rdma_write_ should be called before calling _rdma_accept_, otherwise the notification sent by the client may not be received in time, leading to a block in the server.

**Comment by hellofinch** [comment]
yes! You are right. I follow your suggestion and it worked. If I don't change the order. The error will pop up in client and the server blocked.
